homeassistant/components/homekit/type_sensors.py

```diff
@@ -72,6 +71,7 @@ class SI(NamedTuple):
     format: Callable[[bool], int | bool]
 
 
+DEVICE_CLASS_CO2: Final = "carbon_dioxide"
```

Comment:
Based on https://github.com/home-assistant/architecture/discussions/690, CO2 is available as a device_class in sensor entities, but not in binary_sensor entities.

I have created a local constant, but maybe this shouldn't be using the global constants anyway?

Reply:
cc @bdraco as code-owner for `homekit`.